src/TestSuite/TestCase.php

```diff
@@ -197,21 +197,22 @@ public function loadFixtures()
      * Useful to test how plugins being loaded/not loaded interact with other
      * elements in CakePHP or applications.
      *
-     * @param array $plugins list of Plugins to load
+     * @param array $plugins List of Plugins to load.
      * @return \Cake\Http\BaseApplication
      */
     public function loadPlugins(array $plugins = [])
     {
+        /** @var \Cake\Http\BaseApplication $app */
```

Comment:
after adding this line you can remove 
https://github.com/cakephp/cakephp/blob/bd2cee377d9b63f01a0a6cd443e580adfb588a10/phpstan.neon#L27 line

Reply:
@saeideng done, thanks for pointing out.